src/main/kotlin/com/example/todolist/domain/todolist/presentation/data/response/ToDoListResponse.kt

```diff
@@ -0,0 +1,9 @@
+package com.example.todolist.domain.todolist.presentation.data.response
+
+import com.example.todolist.domain.member.presentation.data.response.MemberResponse
+
+data class ToDoListResponse(
+    val title:String,
+    val content:String,
+    val member: MemberResponse
```

Comment:
```suggestion
    val writer: MemberResponse
```
추후에 writer로 변경한다면 이렇게 변경하는게 좋아보입니다.